src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -418,15 +429,104 @@ private async Task<JObject> GetLocalSettingsJsonAsJObjectAsync()
         public override async Task RunAsync()
         {
             await PreRunConditions();
+            var isVerbose = VerboseLogging.HasValue && VerboseLogging.Value;
 
             var isCurrentProcessNet6Build = RuntimeInformation.FrameworkDescription.Contains(Net6FrameworkDescriptionPrefix);
-            if (isCurrentProcessNet6Build && ShouldLaunchInProcNet8AsChildProcess() && await IsInProcNet8Enabled())
+            if (SetHostRuntime != null)
+            {
+                if (SetHostRuntime == "default")
+                {
+                    if (isCurrentProcessNet6Build)
+                    {
+                        if (isVerbose)
+                        {
+                            ColoredConsole.WriteLine(VerboseColor("Selected out-of-process host"));
+                        }
+                        await StartHostAsChildProcessAsync(true);
```

Comment:
Consider adding the argument name when passing boolean values directly, to improve readability.

Reply:
added!